---
input_file: "pcd_images/aubo_i10/aubo_cloud_out.pcd"
output_file: "pcd_images/aubo_i10/aubo_cloud_out_filtered.pcd"
target_file: "pcd_images/shape1_shape2/shape1.pcd"
save_output: true
translate_output: false
automatic_bounds: false
use_clustering: true

filter_cloud/filter_box: [-1.0, 1.0, -1.0, 0.0, 0.005, 0.5] # bounding box limits
filter_cloud/voxel_leaf_size: 0.001                           # voxel leaf size
